### PR TITLE
GitHub Usecase Implementation

### PR DESCRIPTION
**Pull request by fionnfuchs**

### CODE REVIEW

**Comment by fionnfuchs** [comment]
The tests that fail are controller tests are fine locally for me (both integration and mock). I don't know what that is about...

File: usecase/github/github.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+                    return Reply({'message': "Okay I will ignore the issue."})
+            if self.state == 1:
+                if "yes" in message:
+                    self.state = 0
+                    issue = False
+                    return Reply({'message': "Okay I will set an event in your calendar at that time :)"})
+                if "no" in message:
+                    self.state = 0
+                    issue = False
+                    return Reply({'message': "Okay I maybe you want to look for a time yourself :)"})
```
Comment (mariushegele) [inline usecase/github/github.py]: not an english sentence :)

File: usecase/github/github.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+            for nf in notifications:
+                if nf not in self.seen_notifications:
+                    self.seen_notifications.append(nf)
+                    self.issue = True
+                    self.dispatch_proactive_notification_message(nf)
+                    break
+    
+    def dispatch_proactive_notification_message(self,notification):
+        notification = Notification('New Github Issue!')
+        notification.add_message('What should I do about this new Github Issue? Open Buerro PDA and tell me!')
```
Comment (mariushegele) [inline usecase/github/github.py]: A message is what will be displayed in a chat bubble. So buerro is already open at that point.

File: usecase/github/test/test_usecase.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        self.use_case.set_services(
+            todoist_service=self.todoist_service,
+            calendar_service=self.calendar_service,
+            github_service = self.github_service,
+        )
+    
+    def test_usecase_should_start_unfinished(self):
+        self.assertFalse(self.use_case.finished)
+    
+    def test_usecase_handles_mock_notification(self):
```
Comment (mariushegele) [inline usecase/github/test/test_usecase.py]: you can also mock the notification handler's push:

```py
from unittest.mock import patch

@patch.object(NotificationHandler, 'push')
def test_proactivity(self, mock_notification):

    self.use_case.trigger_proactive_usecase()
    mock_notification.assert_called_once()

```

**Pull request merged**